GVRf/Framework/jni/objects/components/render_data.h

```diff
@@ -85,13 +120,19 @@ class RenderData: public Component {
         rendering_order_ = rendering_order;
     }
 
-    bool cull_test() const {
-        return cull_test_;
-    }
+    bool cull_face(int pass = 0) const {
+    	if (pass >= 0 && pass < render_pass_list_.size()) {
+    		return render_pass_list_[pass]->cull_face();
+    	}
 
-    void set_cull_test(bool cull_test) {
-        cull_test_ = cull_test;
-    }
+    	return nullptr;
+	}
+
```

Comment:
The format for this file looks weird to me.

Please make sure that in Preferences->C++->Code Style. Pick K&R[built-in] profile, change tab policy from "Tab only" to "Spaces only". And do Ctrl+Alt+F for each file you modified.

Thanks in advance.